### DISCUSSION TITLE
Reconfigure Network Interfaces

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#4286] [Unsupported Versions] [answered]
I have the very latest version that I cloned from Github today. I would like to go back and change the settings for my network interfaces. Where are the files that contain this info? Or is there a script to run? so-setup-network will take me through the whole installation.

**Comment by b4b857f6ee** [comment, marked as answer]
not support yet.

**Reply by iqworks** [reply]
I know this post was in May, and I see “not supported yet”? not sure what that means?   

  I am using Wazuh 2.3.70 with centOS 7.  
  I am trying to see the network interfaces. 
   With this command “cat /etc/sysconfig/network-scripts/ifcfg-* ” I see :
![image](https://user-images.githubusercontent.com/5569263/133909887-d41ae288-31d0-434d-9374-b157ccb834e3.png)
![image](https://user-images.githubusercontent.com/5569263/133909893-a909028e-2a3a-42eb-8800-743108ec93c2.png)
![image](https://user-images.githubusercontent.com/5569263/133909894-5e43abb6-53fc-46b6-8353-ed5bb02fb5dd.png)
![image](https://user-images.githubusercontent.com/5569263/133909902-35c10090-95a9-4039-86ca-228e7eafee0e.png)
![image](https://user-images.githubusercontent.com/5569263/133909904-da26900a-c294-43e0-a5f8-3ea8a67cfc92.png)

But when I look here “ https://docs.securityonion.net/en/16.04/network-configuration.html ”, I see a reference made to “/etc/network/interfaces”. 
 So, I get confused about the proper and/or correct way to look at my interfaces and their configurations?. 
 And what does “Yes, configure /etc/network/interfaces” in the setup mean, I do not see this in the setup?
 What does “If for some reason you need to configure /etc/network/interfaces manually, you can do the following.” Mean? What command do you use to access “/etc/network/interfaces manually”?

 Which set of commands  do I use in my version of security onion?  The one that responds to “/etc/network/” or the one that responds to “/etc/sysconfig/network-scripts” to work with my NIC interfaces?

I was looking for how many interfaces security onion needs, is it only 2? Allow and manage?
 It seems like I only need 2 interfaces for my security onion VM, one for management and one for allow (or sniffing), is that all I need to use SO?

Should my SO VM be using my NAT (198.162.80.0) as the management NIC IP in my VMware 16 pro? 
should my allow (sniffing) IP be one of my VMnet’s that are a simple custom VMnet that is not used by anything or anybody else?

I see it says security onion works best with wired connections, does my SO VM need to consider a wired connection or wireless connection to access the SOC from my home lab setup? My windows 10 uses wireless to access the web, shouldn’t my NAT and bridge VMnet’s automatically use my windows 10 wireless? 

  if these are to many questions to ask at the same time, please, let me know so I can create a new posts for each issue. 
 
 I just hope I am not breaking the rules by asking to many related questions in one post. 

 thanks, i am still learning

**Reply by dougburks** [reply]
@iqworks From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.
